### PR TITLE
Generate report with `remediation_suggestion`

### PR DESCRIPTION
**Pull request by shreyamalviya**
# What does this PR do?

Fixes a part of #2857 

## PR Checklist
* [ ] Have you added an explanation of what your changes do and why you'd like to include them?
* [ ] Is the TravisCI build passing?
* [ ] Was the CHANGELOG.md updated to reflect the changes?
* [ ] Was the documentation framework updated to reflect the changes?
* [ ] Have you checked that you haven't introduced any duplicate code?

## Testing Checklist

* [ ] Added relevant unit tests?
* [x] Have you successfully tested your changes locally? Elaborate:
    > Tested by running some manual tests and verifying
* [ ] If applicable, add screenshots or log transcripts of the feature working

image
image

### CODE REVIEW

**Comment by shreyamalviya** [comment]
image

**Review by ilija-lazoroski — APPROVED** [review]

**Comment by shreyamalviya** [comment]
Tested that non-plugin issues are shown as expected:
image

File: monkey/monkey_island/cc/ui/src/components/report-components/SecurityReport.js [hunk 1]
```diff
@@ -503,29 +285,62 @@ class ReportPageComponent extends AuthComponent {
   generateReportFooter() {
     return (
       <div id='footer' className='text-center' style={{ marginTop: '20px' }}>
-        For questions, suggestions or any other feedback
-        contact: <a href='mailto://labs@guardicore.com' className='no-print'>labs@guardicore.com</a>
-        <div className='force-print' style={{ display: 'none' }}>labs@guardicore.com</div>
+        For questions, suggestions, or any other feedback,
+        contact <a href='mailto://labs@guardicore.com' className='no-print'>labs@guardicore.com</a>
+        <div className='force-print' style={{ display: 'none' }}>labs@guardicore.com</div>.
```
Comment (VakarisZ) [inline monkey/monkey_island/cc/ui/src/components/report-components/SecurityReport.js:290]: This probably should be changed to `support@infectionmonkey.com`, but let's not cram more changes into this PR. Typos in the report have nothing to do with plugins.

Comment (shreyamalviya) [reply monkey/monkey_island/cc/ui/src/components/report-components/SecurityReport.js:290]: Right, I'll open another PR with these changes.

**Review by VakarisZ — APPROVED** [review]

File: monkey/monkey_island/cc/ui/src/components/report-components/SecurityReport.js (hunk 1 above)
Comment (shreyamalviya) [reply monkey/monkey_island/cc/ui/src/components/report-components/SecurityReport.js]: https://github.com/guardicore/monkey/pull/2899

**Pull request merged**